src/rules/apigw/APIGWRequestValidation.ts

```diff
@@ -27,6 +29,36 @@ export default Object.defineProperty(
           }
         }
       }
+      if (node.bodyS3Location) {
+        const assetOutdir = NagRules.resolveResourceFromInstrinsic(
+          node,
+          Stage.of(node)?.assetOutdir
+        );
+        const assetPath = NagRules.resolveResourceFromInstrinsic(
+          node,
+          node.getMetadata('aws:asset:path')
+        );
+        console.log(assetOutdir, assetPath);
```

Comment:
Need to remove the log statement

Reply:
Sorry, I must have forgotten to remove that. It should be fixed now.